example/65_grouped_conv_fwd_convscale/convnd_fwd_convscale_common.hpp

```diff
@@ -0,0 +1,337 @@
+    copy(in_g_n_c_wis_desc.GetStrides(), a_g_n_c_wis_strides);
+    copy(wei_g_k_c_xs_desc.GetLengths(), b_g_k_c_xs_lengths);
+    copy(wei_g_k_c_xs_desc.GetStrides(), b_g_k_c_xs_strides);
+    copy(d0_g_n_k_wos_desc.GetLengths(), d0_g_n_k_wos_lengths);
+    d0_g_n_k_wos_strides.fill(0); // d0 is a scalar
+    copy(d1_g_n_k_wos_desc.GetLengths(), d1_g_n_k_wos_lengths);
+    d1_g_n_k_wos_strides.fill(0); // d1 is a scalar
+    copy(d2_g_n_k_wos_desc.GetLengths(), d2_g_n_k_wos_lengths);
+    d2_g_n_k_wos_strides.fill(0); // d2 is a scalar
+    copy(out_g_n_k_wos_desc.GetLengths(), e_g_n_k_wos_lengths);
+    copy(out_g_n_k_wos_desc.GetStrides(), e_g_n_k_wos_strides);
+    copy(conv_param.conv_filter_strides_, conv_filter_strides);
+    copy(conv_param.conv_filter_dilations_, conv_filter_dilations);
+    copy(conv_param.input_left_pads_, input_left_pads);
+    copy(conv_param.input_right_pads_, input_right_pads);
+
+    // do Conv
+    auto conv    = DeviceConvNDFwdInstance{};
+    auto invoker = conv.MakeInvoker();
+    auto argument =
+        conv.MakeArgument(in_device_buf.GetDeviceBuffer(),
+                          wei_device_buf.GetDeviceBuffer(),
+                          std::array<const void*, 3>{d0_device_buf.GetDeviceBuffer(),
+                                                     d1_device_buf.GetDeviceBuffer(),
+                                                     d2_device_buf.GetDeviceBuffer()},
+                          out_device_buf.GetDeviceBuffer(),
+                          a_g_n_c_wis_lengths,
+                          a_g_n_c_wis_strides,
+                          b_g_k_c_xs_lengths,
+                          b_g_k_c_xs_strides,
+                          std::array<std::array<ck::index_t, NDimSpatial + 3>, 3>{
+                              d0_g_n_k_wos_lengths, d1_g_n_k_wos_lengths, d2_g_n_k_wos_lengths},
+                          std::array<std::array<ck::index_t, NDimSpatial + 3>, 3>{
+                              d0_g_n_k_wos_strides, d1_g_n_k_wos_strides, d2_g_n_k_wos_strides},
+                          e_g_n_k_wos_lengths,
+                          e_g_n_k_wos_strides,
+                          conv_filter_strides,
+                          conv_filter_dilations,
+                          input_left_pads,
+                          input_right_pads,
+                          in_element_op,
+                          wei_element_op,
+                          out_element_op);
+
+    if(!conv.IsSupportedArgument(argument))
+    {
+        throw std::runtime_error(
+            "wrong! device_conv with the specified compilation parameters does "
+            "not support this Conv problem");
+    }
+
+    float avg_time = invoker.Run(argument, StreamConfig{nullptr, time_kernel});
+
+    std::size_t ds_size   = 3; // 3 element-wise scale multipliers
+    std::size_t flop      = GetFlops<NDimSpatial>(e_g_n_k_wos_lengths, b_g_k_c_xs_lengths, ds_size);
```

Comment:
We should take into account elementwise flops

Reply:
Sure, we take into account, here is how we calculate it in GetFlops: G * N * C * <output spatial lengths product> * (2 * K * <filter spatial lengths product> + <number of scale factors>). Overall, elementwise op adds ds_size*output_pixels flops.